packages/openneuro-app/src/scripts/resources/kibana.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Extract kibana deployment uri from ELASTICSEARCH_CLOUD_ID (prod/staging)
+ *   or use localhost url.
+ * For localhost url to be available, the kibana service must be enabled (docker-compose).
+ */
+const getKibanaURI = () => {
+  if (process.env.ELASTICSEARCH_CLOUD_ID) {
+    const ELASTICSEARCH_CLOUD_ID = process.env.ELASTICSEARCH_CLOUD_ID
+    const base64 = /:(.+?==)$/.exec(ELASTICSEARCH_CLOUD_ID)[1]
+    const decoded = atob(base64)
+    const deploymentId = /\$.+?\$(.+?)$/.exec(decoded)[1]
+    return `${deploymentId}.us-east-1.aws.found.io:9243/app/discover#/`
```

Comment:
The hostname here is also in the cloud ID, to make this more robust to deployment (I could see it breaking if the ElasticSearch setup was rebuilt) it should use that.